perf/istio-install/values.yaml

```diff
@@ -169,12 +183,12 @@ certmanager:
 
 
 nodeagent:
-  enabled: false
+  enabled: true
```

Comment:
does this mean workload identity provision for sidecar switch to sds in perf-test as well ?

Reply:
yes, I would like to looking ahead.
Until your PR merges, we will need to keep controlPlaneSecurity=off.
But that should be shortly.